package.json

```diff
@@ -305,7 +303,6 @@
     "netlify-redirector": "^0.3.1",
     "node-fetch": "^2.6.0",
     "node-version-alias": "^1.0.1",
-    "omit.js": "^2.0.2",
```

Comment:
Used destructuring instead of omit.js

```diff
-x = omit(config.build, ['publish', 'publishOrigin']) }
+// eslint-disable-next-line no-unused-vars
+const { publish, publishOrigin, ...x } = config.build
```